### ISSUE TITLE
Web3Geth Miner Start always returns false.

### ISSUE THREAD

**Issue by jcalixv** [Nethereum/Nethereum#127]
Hello, first of all, thank you very much for this amazing utility.

I'm beginning to learn about nethereum, and I want to use this library, cause we mainly work in .Net.
I want to transfer some ether from SRC account to Dest Account, almost everything works as expected:

            var web3 = new Web3(@"http://localhost:8545");
            var web3Geth = new Web3Geth(web3.Client);
            
            var ethSendTransation = new EthSendTransaction(_web3.Client);
            var transactionInput = new TransactionInput
            {
                From = _mainAccount,
                To = _adminAccount,
                Value = new HexBigInteger(UnitConversion.Convert.ToWei(2))
            };
           var result = await web3Geth.Miner.Start.SendRequestAsync(1);
            if (result != true) throw new Exception("Could not start mining");

//**The Start Mining Always return False, but the miner do Starts.**

            string txHash = String.Empty;
            var unlockAccount = await _web3.Personal.UnlockAccount.SendRequestAsync(_mainAccount,   _password, 600);
            if (unlockAccount)
            {
                txHash = await ethSendTransation.SendRequestAsync(transactionInput);
                Console.WriteLine(txHash);
            }
            if (!string.IsNullOrWhiteSpace(txHash))
            {
                var txReceipt = await TransactionReceipt(_web3, txHash);
                Console.WriteLine(JsonConvert.SerializeObject(txReceipt));
            }

            result = await web3Geth.Miner.Stop.SendRequestAsync();
            if (result != true) throw new Exception("Could not stop mining");

That line is the one that is not working, or I'm doing something wrong.
any help in the right direction would be most appreciated.
Thanks.

Note.
Nethereum.Portable 2.0.0-rc3
Net 4.6.1, UnitTest Project
VS2017

**Comment by juanfranblanco** [comment]
**The Start Mining Always return False, but the miner do Starts.**

This is rather confusing (and interesting)

I assume you are using Geth as your Ethereum Client, what version are you using?

Based on that assumption I also assume that you have enabled mining, when launching geth, if not it will throw an exception.

Can you enable Geth with "console" (last argument) and "verbosity 0", and type "miner.start()" and see the return value?

**Comment by jcalixv** [comment]
Hello, thank you very much for the fast reply.

Like you said, I'm using Geth v1.6.1 stable.
I'm on private dev chain.

This is the command I used to start:
geth --datadir "L:\chains\usd" --rpc --rpcport 8545  --rpcaddr 192.168.100.13 --rpcapi "eth,web3,personal,net,miner,admin,debug" --dev console 2>> L:\chains\usd.log

The miner.start() command returns null;

I did run the command on another node, private chain (not dev)  and
The miner.start() command returns null too.

So thanks to your help and this tests, this is a Geth problem.
So sorry. Ill closed this.
Thank you very much for your help and this great library.
